<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ADD MEMBER</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>LM Roman 17</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>REGISTER MEMBER FROM PREVIOUS RECORDS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_to_members">
   <property name="geometry">
    <rect>
     <x>886</x>
     <y>30</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>GO TO MEMBERS PAGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="home_btn">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>30</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>HOME</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_member">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE MEMBER</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cantarell</family>
    </font>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>FAMILY &amp; EMERGENCY CONTACTS</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>150</y>
     <width>551</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="ttc">
      <property name="text">
       <string>TOTAL RECORDED CONTRIBUTIONS</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="total_contribs"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="tOTOALDEBTSANDLOANSLabel">
      <property name="text">
       <string>TOTAL DEBTS AND LOANS</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="total_loans"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="tOTALOVERDUEPAYMENTSLabel">
      <property name="text">
       <string>TOTAL OVERDUE PAYMENTS</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="total_overdue"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="totalArrearsLabel">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total Arrears</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="total_arrears"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>108</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FINANCIAL DETAILS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_new_btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>500</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD NEW</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>328</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>OTHER DETAILS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>370</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>PROFILE PICTURE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upload_button">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>370</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>UPLOAD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>419</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>HEALTH DETAILS</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="health_details_box">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>410</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pOSITIONROLELabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>114</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>POSITION/ ROLE</string>
   </property>
  </widget>
  <widget class="QLabel" name="pHONENUMBER1Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>126</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>PHONE NUMBER 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pHONENUMBER2LineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>635</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="fULLNAMELineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>635</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nATIONALIDLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>635</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="pOSITIONROLEComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>635</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Member</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Admin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Secretary</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Treasurer</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="dATEOFBIRTHLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>106</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>DATE OF BIRTH</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pHONENUMBER1LineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>635</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sTATUSLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>422</y>
     <width>53</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>STATUS</string>
   </property>
  </widget>
  <widget class="QLabel" name="hOMEADDRESSLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>HOME ADDRESS</string>
   </property>
  </widget>
  <widget class="QLabel" name="fULLNAMELabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>FULL NAME</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sTATUSComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>635</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ACTIVE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>INACTIVE</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="dATEOFBIRTHDateEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>635</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pHONENUMBER2Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>128</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>PHONE NUMBER 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="dATEJOINEDLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>DATE JOINED</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hOMEADDRESSLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>635</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nATIONALIDLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>NATIONAL ID</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dATEJOINEDDateEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>635</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>761</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="family_form"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
